rnd/market/Dockerfile

```diff
@@ -25,19 +26,31 @@ ENV POETRY_VERSION=1.8.3 \
     PATH="$POETRY_HOME/bin:$PATH"
 RUN pip3 install poetry
 
-COPY rnd/market /app/rnd/market
+FROM server_base AS server_dependencies
+
+RUN mkdir -p /app/autogpt
+RUN mkdir -p /app/forge
+RUN mkdir -p /app/rnd/autogpt_libs
+RUN mkdir -p /app/rnd/market
 COPY rnd/autogpt_libs /app/rnd/autogpt_libs
 
+COPY rnd/market/poetry.lock rnd/market/pyproject.toml /app/rnd/market/
+
 WORKDIR /app/rnd/market
 
-# Install dependencies
 RUN poetry install --no-interaction --no-ansi
 
+FROM server_dependencies AS server_prisma
```

Comment:
Why the extra stages? (`server_dependencies`, `server_prisma`)

The way they're being used now gives the same result as having just a single stage.